### PR TITLE
Add pageview

### PR DESCRIPTION
**Pull request by OnderUlasKilic**
Add pageview

### CODE REVIEW

File: lib/101/page_view_learn.dart [hunk 1]
```diff
@@ -0,0 +1,75 @@
+// ignore_for_file: unused_import
```
Comment (BBarisKilic) [inline lib/101/page_view_learn.dart]: Remove this unnecessary line.

File: lib/101/page_view_learn.dart [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+      _correntPageIndex = index;
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      floatingActionButton: Row(
+        // mainAxisSize: MainAxisSize.min,
+
```
Comment (BBarisKilic) [inline lib/101/page_view_learn.dart]: Remove comment along with the space.

File: lib/101/page_view_learn.dart [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+  const PageViewLearn({super.key});
+
+  @override
+  State<PageViewLearn> createState() => _PageViewLearnState();
+}
+
+class _PageViewLearnState extends State<PageViewLearn> {
+  final _pageController = PageController(viewportFraction: .7);
+
+  int _correntPageIndex = 0;
```
Comment (BBarisKilic) [inline lib/101/page_view_learn.dart]: Fix typo.

```suggestion
  int _currentPageIndex = 0;
```

File: lib/101/statefull_learn.dart [hunk 4]
```diff
@@ -50,6 +59,7 @@ class _IncrementButton extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    print("burda");
```
Comment (BBarisKilic) [inline lib/101/statefull_learn.dart]: Remove this print. (You should avoid print calls in production code.)

File: lib/product/counter_hello_button.dart [hunk 5]
```diff
@@ -0,0 +1,32 @@
+// ignore_for_file: unused_import
```
Comment (BBarisKilic) [inline lib/product/counter_hello_button.dart]: Remove this unnecessary line.

File: lib/product/counter_hello_button.dart [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+  const CounterHelloButton({super.key});
+
+  @override
+  State<CounterHelloButton> createState() => _CounterHelloButtonState();
+}
+
+class _CounterHelloButtonState extends State<CounterHelloButton> {
+  int _counterCustom = 0;
+  final String _wellcomeTitle = "Merhaba";
+  void _updateCounter() {
```
Comment (BBarisKilic) [inline lib/product/counter_hello_button.dart]: Add space between variables and methods.

```suggestion
  final String _wellcomeTitle = "Merhaba";
  
  void _updateCounter() {
```

File: lib/product/counter_hello_button.dart [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return ElevatedButton(
+      onPressed: () {
+        setState(() {
+          ++_counterCustom;
+        });
+      },
```
Comment (BBarisKilic) [inline lib/product/counter_hello_button.dart]: (Optional) You may prefer to use expression body as we discussed.

```suggestion
      onPressed: () => setState(() => ++_counterCustom),
```

**Review by BBarisKilic — CHANGES_REQUESTED** [review]
There are some small changes required, but besides looks fine.

File: lib/101/page_view_learn.dart (hunk 1 above)
Comment (OnderUlasKilic) [reply lib/101/page_view_learn.dart]: Done.

File: lib/101/page_view_learn.dart (hunk 2 above)
Comment (OnderUlasKilic) [reply lib/101/page_view_learn.dart]: Done.

File: lib/101/page_view_learn.dart (hunk 3 above)
Comment (OnderUlasKilic) [reply lib/101/page_view_learn.dart]: Done.

File: lib/101/statefull_learn.dart (hunk 4 above)
Comment (OnderUlasKilic) [reply lib/101/statefull_learn.dart]: Done.

File: lib/product/counter_hello_button.dart (hunk 5 above)
Comment (OnderUlasKilic) [reply lib/product/counter_hello_button.dart]: Done.

File: lib/product/counter_hello_button.dart (hunk 6 above)
Comment (OnderUlasKilic) [reply lib/product/counter_hello_button.dart]: Done.

File: lib/product/counter_hello_button.dart (hunk 7 above)
Comment (OnderUlasKilic) [reply lib/product/counter_hello_button.dart]: Done.

**Review by BBarisKilic — APPROVED** [review]

**Pull request merged**